akismet-privacy-policies.php

```diff
@@ -86,6 +110,30 @@ public static function get_object() {
 		return self::$classobj;
 	}
 
+	/**
+	 * Initialize and translate $this->notice, $this->error_message
+	 *
+	 * @since x.x.x
+	 * @access public
+	 *
+	 * @return void
+	 */
+	public function translate_strings() {
+		// dummy translation, only there to let poedit recognize the text...
+		__( 'I accept that my given data and my IP address is sent to a server in the USA only for the purpose of spam prevention through the <a href="http://akismet.com/">Akismet</a> program.<br /><a href="https://akismet.com/gdpr/">More information on Akismet and GDPR</a>.', 'akismet-privacy-policies' );
+    __( '<p><strong>Attention:</strong> You have not accepted our privacy disclaimer.</p>', 'akismet-privacy-policies' );
+		if ( $this->translation != 'en_US' ) {
+			$this->notice = $this->mo->translate( 'I accept that my given data and my IP address is sent to a server in the USA only for the purpose of spam prevention through the <a href="http://akismet.com/">Akismet</a> program.<br /><a href="https://akismet.com/gdpr/">More information on Akismet and GDPR</a>.', 'akismet-privacy-policies' );
+		} else {
```

Comment:
The else control is not necessary, so we can remove them.

Reply:
I think this is a special case. If I leave out the `else` branch the English template strings are not displayed. The English template strings are not held in a `.po` file - they're defined in the php source. Thus, I'm assigning the untranslated version of the template string.
The `$this->mo->translate()` may seem a bit awkward. The idea behind is, that, if e.g. the admin interface's language has been switched to English, the template strings should still appear in the current translation language.